crates/holochain_conductor_api/src/config/conductor.rs

```diff
@@ -126,7 +126,7 @@ pub struct ConductorConfig {
     pub admin_interfaces: Option<Vec<AdminInterfaceConfig>>,
 
     /// Optional config for the network module.
-    #[serde(default)]
+    #[serde(default = "default_network_config")]
```

Comment:
This will only get set if the user doesn't provide a config. My understanding of what we discussed is that the user's config would get overridden unless they enabled the associated feature.